### PR TITLE
mdns_responder - Clarify what code in the sample is necessary for mdn…

### PR DESCRIPTION
**Pull request by chmorgan**
…s support vs. what code is unrelated to mdns

### CODE REVIEW

File: samples/net/mdns_responder/src/main.c [hunk 1]
```diff
@@ -14,8 +14,14 @@ LOG_MODULE_REGISTER(net_mdns_responder_sample, LOG_LEVEL_DBG);
...
 
-/* Note that this application does not do anything itself.
- * It is just a placeholder for waiting mDNS queries.
+/*
+ * Note that mdns support requires no application interaction with zephyr,
+ * beyond optional runtime hostname configuration calls and setting
+ * CONFIG_MDNS_RESPONDER=y.
+ *
+ * The service() function provides an echo server to make it possible to
+ * verify that the ip address resolved by mdns is the system that this
```
Comment (rlubos) [inline samples/net/mdns_responder/src/main.c:23]:
```suggestion
 * verify that the IP address resolved by mDNS is the system that this
```

**Review by rlubos — COMMENTED** [review]
Please follow https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-message-guidelines wrt commit message format. Specifically, plase use `samples: net: mdns_responder: ` prefix in the commit title.

**Review by jukkar — COMMENTED** [review]
The change LGTM, you need fix the commit msg as the checker complains

**Comment by rlubos** [comment]
Compliance is still not happy:
>  1: UC5 Commit title exceeds max length (95>75): "samples: net: mdns_responder: Clarify code in the sample for mDNS support vs. unrelated to mDNS"
1: UC6 Commit message body is empty, should at least have 1 line(s).

**Comment by chmorgan** [comment]
@rlubos that's a pretty short subject line requirement. I believe his latest push fixes those remaining issues.

**Review by rlubos — APPROVED** [review]

**Review by jukkar — APPROVED** [review]

**Pull request merged**